projects_info.csv

```diff
@@ -18,7 +18,7 @@ Y | saga | 9.8.0 | Y | Y | Desktop GIS | | | saga-gis | https://saga-gis.sourcef
 # OSGeo Projects | | | | | | | | | | | |
 Y | geomoose | 3.10.1 | Y | Y | Browser Facing GIS | | Web GIS Portal | geomoose | https://www.geomoose.org | OSGeo_project | GeoMoose | Y
 Y | geonode | 3.3.2 | Y | Y | Browser Facing GIS | | Geospatial Content Management System | geonode | https://geonode.org | OSGeo_project | GeoNode | Y
-Y | mapbender | 4.1.0 | Y | Y | Browser Facing GIS | | Geoportal Framework | mapbender | https://mapbender.org | OSGeo_project | Mapbender | Y
+Y | mapbender | 4.1.1 | Y | Y | Browser Facing GIS | | Geoportal Framework | mapbender | https://mapbender.org | OSGeo_project | Mapbender | Y
```

Comment:
Looks like it should be 4.0.2
https://github.com/OSGeo/OSGeoLive/blob/master/CHANGES.txt#L18

Reply:
https://github.com/OSGeo/OSGeoLive/commit/b6c57ac538922613d8a64592d59a3350d6fecd1d